### PR TITLE
Updating Wazuh-DB IT documentation with agent's CVE table commands

### PR DESCRIPTION
**Pull request by pereyra-m**
|Related issue|
|---|
|1116|

## Description

This PR creates the Wazuh-DB IT documentation section, but only adds the information related to the new file **test_wazuh_db/data/agent_messages.yaml**.

Also, some internal broken links were fixed. 

## Tests

- [x] The test is documented in wazuh-qa/docs

### CODE REVIEW

File: docs/tests/integration/test_wazuh_db/index.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
 # Overview
 
+**Wazuh-db** is the daemon in charge of the databases with all the Wazuh persistent information, exposing a socket so receive requests and provide information.
```
Comment (palaciosjeremias) [inline docs/tests/integration/test_wazuh_db/index.md:3]: Typo: **so** -> **to**

File: docs/tests/integration/test_wazuh_db/index.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
 
+## Tiers
+### Tier 0
+#### Test wazuh_db
+
+These tests check the Wazuh-db commands syntax, valid responses and error messages in different circumstances. They are divided in **.yaml** configuration files:
+
+- **agent_messages.yaml**: tests for agents in general, currently only for Agents' CVEs table
```
Comment (palaciosjeremias) [inline docs/tests/integration/test_wazuh_db/index.md:11]: I think we can avoid this part: _currently only for Agents' CVEs table_

i.e.:
_tests for agents DB, general commands._

File: docs/tests/integration/test_wazuh_db/index.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+- **fim_messages.yaml**: tests for agents, but only for FIM module
```
Comment (palaciosjeremias) [inline docs/tests/integration/test_wazuh_db/index.md:12]: Maybe we can express this like:
_tests for agents DB, FIM module commands._

File: docs/tests/integration/test_wazuh_db/test_wazuh_db.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## Expected behavior
+
+- Fail if `wazuh-db` response is different from output
+## Testing
+
+Tests description according to its classification
+### Checks agent_messages
+
+Currently, only **insert** and **clear** commands for `vuln_cves` table are tested
```
Comment (palaciosjeremias) [inline docs/tests/integration/test_wazuh_db/test_wazuh_db.md:40]: I think if we avoid 
_Currently, only_ part will make this documentation easier to update.
i.e.:  _**insert** and **clear** commands for `vuln_cves` table are tested._

File: docs/tests/integration/test_wazuh_db/test_wazuh_db.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+# Test Wazuh DB
+
+All these tests are meant to send a specific command to the socket (valid or not) and compare the resulting output with the expected result.
```
Comment (DProvinciani) [inline docs/tests/integration/test_wazuh_db/test_wazuh_db.md:3]: Y suggest moving `(valid or not)` before to `to the socket`. In the current location seems like `the socket` could be valid or not, and I think that you are referring to the `command`.

File: docs/tests/integration/test_wazuh_db/test_wazuh_db.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+## Overview
```
Comment (DProvinciani) [inline docs/tests/integration/test_wazuh_db/test_wazuh_db.md:4]: Shouldn't we leave a white line between the previous paragraph and this title?

File: docs/tests/integration/test_wazuh_db/test_wazuh_db.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    output: "ok"
+    stage: "global update-agent-name success"
+```
+
+All similar test are grouped by a name, like `Update commands`. Then, every test has:
+
+- **input**: the command that will be sent to the socket
+- **output**: the expected result
+- **stage**: the name of that particular test
+## Objective
```
Comment (DProvinciani) [inline docs/tests/integration/test_wazuh_db/test_wazuh_db.md:22]: Ditto

**Review by DProvinciani — CHANGES_REQUESTED** [review]
Please, apply changes according to the comments.

File: docs/tests/integration/test_wazuh_db/index.md (hunk 1 above)
Comment (pereyra-m) [reply docs/tests/integration/test_wazuh_db/index.md:3]: Thank you! 
Fixed

File: docs/tests/integration/test_wazuh_db/index.md (hunk 2 above)
Comment (pereyra-m) [reply docs/tests/integration/test_wazuh_db/index.md:11]: Fixed!

File: docs/tests/integration/test_wazuh_db/index.md (hunk 3 above)
Comment (pereyra-m) [reply docs/tests/integration/test_wazuh_db/index.md:12]: OK, fixed!

File: docs/tests/integration/test_wazuh_db/test_wazuh_db.md (hunk 4 above)
Comment (pereyra-m) [reply docs/tests/integration/test_wazuh_db/test_wazuh_db.md:40]: You are right, fixed!

File: docs/tests/integration/test_wazuh_db/test_wazuh_db.md (hunk 5 above)
Comment (pereyra-m) [reply docs/tests/integration/test_wazuh_db/test_wazuh_db.md:3]: It's true, fixed!

File: docs/tests/integration/test_wazuh_db/test_wazuh_db.md (hunk 6 above)
Comment (pereyra-m) [reply docs/tests/integration/test_wazuh_db/test_wazuh_db.md:4]: Fixed!

File: docs/tests/integration/test_wazuh_db/test_wazuh_db.md (hunk 7 above)
Comment (pereyra-m) [reply docs/tests/integration/test_wazuh_db/test_wazuh_db.md:22]: Fixed!

**Review by DProvinciani — APPROVED** [review]
Looks good!

**Review by palaciosjeremias — APPROVED** [review]
LGTM

**Pull request merged**